ynh_add_swap/ynh_add_swap

```diff
@@ -17,6 +17,14 @@ ynh_add_swap () {
 	# Because we don't want to fill the disk with a swap file, divide by 2 the available space.
 	local usable_space=$(( $free_space / 2 ))
 
+ 	SD_CARD_CAN_SWAP=${SD_CARD_CAN_SWAP:-0}
+
+	# Swap on SD card only if it's is specified
+	if ynh_is_main_device_is_sd_card && [ "$SD_CARD_CAN_SWAP" == "0" ]
+	then
+		return
```

Comment:
Add a warning here! And explain that swaping can be force by using `SD_CARD_CAN_SWAP=1`

Reply:
+1